main.tf

```diff
@@ -12,6 +12,28 @@ provider "aws" {
   profile = "codeforpoznan"
 }
 
+data "aws_iam_policy_document" "codeforpoznan_public_policy_document" {
```

Comment:
we should decide if we want to add `_document` suffix everywhere or omit it everywhere. Right now we have both approaches.

Reply:
will remove it